src/uu/false/src/false.rs

```diff
@@ -5,6 +5,15 @@
 //  * For the full copyright and license information, please view the LICENSE
 //  * file that was distributed with this source code.
 
+use clap::{App, AppSettings};
+use uucore::executable;
+
 pub fn uumain(_: impl uucore::Args) -> i32 {
     1
 }
+
+pub fn uu_app() -> App<'static, 'static> {
+    App::new(executable!())
+        .setting(AppSettings::DisableHelpFlags)
+        .setting(AppSettings::DisableVersion)
```

Comment:
`false` and `true` have `--help` and `--version`, but in both `bash` and `fish` (and probably other shells), the coreutils version is not used. I can run it with `/usr/bin/false --help`, for example. However, since the uutils false and true did not have these arguments, I'm also fine with keeping it like this and fixing it in another PR.

Reply:
Oh! I wasn't aware of that for exactly that reason. I'll fix it in this pr